src/js/edu-benefits/5490/config/form.js

```diff
@@ -176,7 +172,7 @@ const formConfig = {
                 'ui:options': {
                   expandUnder: 'view:claimedSponsorService',
                   updateSchema: (data, form) => {
-                    if (_.get(['previousBenefits', 'data', 'previousBenefits', 'view:claimedSponsorService'], form)) {
+                    if (_.get('previousBenefits.data.previousBenefits.view:claimedSponsorService', form)) {
```

Comment:
Yeah, I thought that might be the case too, so I tested it out and it looks like it's alright!